ui/plugins/controller.js

```diff
@@ -88,7 +93,11 @@ treeherder.controller('PluginCtrl', [
                     $scope.eta_abs = Math.abs($scope.job.get_current_eta());
                     $scope.typical_eta = $scope.job.get_typical_eta();
                     $scope.jobRevision = ThResultSetStore.getSelectedJob($scope.repoName).job.revision
-
+                    $scope.job_ids = results[4];
+                    console.log(results);
+                    _.forEach($scope.job_ids, function(job) {
+                        console.log(job[0].value);
+                    });
```

Comment:
<img width="1261" alt="screen shot 2015-11-01 at 9 24 54 am" src="https://cloud.githubusercontent.com/assets/2166227/10866768/1fa2953e-807b-11e5-982c-66ff1410d8cf.png">
So, I pick up another example. The project name is **mozilla-inbound** and the job id is **16628134**. The link of job id is https://treeherder.mozilla.org/api/project/mozilla-inbound/performance/data/?job_id=16628134.